packages/bot-engine/src/app.js

```diff
@@ -0,0 +1,15 @@
+import DBot         from './scratch/dbot';
```

Comment:
Are we able to have nicer paths for imports? i.e. not import everything from `@deriv/bot-engine`, but something like `@deriv/bot-engine/utils`?

Reply:
@aaron-binary yes we can and we will in next PR.